tests/Api/Groups/GroupTest.php

```diff
@@ -65,4 +66,17 @@ public function testRetrieveGroupInformation()
 
         $this->assertInstanceOf(RetrieveGroupResponse::class, $response);
     }
+    
+    /**
+     * 
+     */
```

Comment:
```suggestion
```

Reply:
We need Integration group here for this test:

```suggestion
    /**
     * @group Integration
     */
```

Reply:
@Vitexus we need to include a test under `Integration` group whenever we call real api. See example above.